src/host/_stream.cpp

```diff
@@ -515,6 +521,10 @@ constexpr unsigned int LOCAL_BUFFER_SIZE = 100;
                             }
                             else
                             {
+                                // WCL-NOTE: We should never hit this.
+                                // WCL-NOTE: 1. Normal characters are handled via the early check for IS_GLYPH_CHAR
+                                // WCL-NOTE: 2. Control characters are handled via the CtrlChar label (if WC_PRINTABLE_CONTROL_CHARS is on)
+                                // WCL-NOTE:    And if they are control characters they will trigger the C1_CNTRL check above.

```

Comment:
This was observed by fuzzing for ~10 minutes with `WC_PRINTABLE_CONTROL_CHARS` on or off.